cfgov/unprocessed/css/organisms/secondary-navigation.less

```diff
@@ -94,6 +94,9 @@
         }
 
         .o-expandable_content {
+            // These two !important values override basic expandable styling,
+            // because these don't function like expandables on med+ screens.
+            display: block !important;
```

Comment:
Maybe instead of this we should add a separate block so this and the "read more" edit are consistent:

```
.o-expandable_content__collapsed {
            // These two !important values override basic expandable styling,
            // because these don't function like expandables on med+ screens.
            display: block !important;
}

.o-expandable_content {
            max-height: 100% !important;
            padding: 0;
…
}
```

Reply:
I'm not sure I follow. Did your code block get messed up?

Reply:
Like create a new block that targets the `.o-expandable_content__collapsed`, not `o-expandable_content`. That will still target the same element on the page, but that way both of the edits you have in this PR will be overriding the same class.

Reply:
Ah! Sounds good.

Reply:
Also realizing now that it needs to be targeted at `@bp-sm-min` to avoid _also_ taking precedence on mobile devices.

Reply:
Updated in c00bfab.

Reply:
You're targeting `o-expandable_content` instead of `o-expandable_content__eollapsed` in the override, because the [JS targets the content class](https://github.com/cfpb/design-system/blob/master/packages/cfpb-expandables/src/Expandable.js#L57). That makes sense.